projects/simulation/human_dataset_generation/README.md

```diff
@@ -0,0 +1,48 @@
+# Human-Data-Generation-Framework
+
+This folder contains the code for generating the data described in "Efficient Realistic Data Generation Framework leveraging Deep Learning-based Human Digitization"
+
+## Download and reformat the CityScapes dataset
+
+1. Download the 3D human models
+
+```
+chmod +x download_models.sh
+./download_models.sh
+```
+
+2. Download the CityScapes dataset from www.cityscapes-dataset.net <br />
```

Comment:
The correct spelling seems to be (based on their website)
```suggestion
2. Download the Cityscapes dataset from www.cityscapes-dataset.net <br />
```

Could you please also remove the `<br/>`. They should not be needed.